tools/noise/subr.sh

```diff
@@ -446,11 +446,12 @@ export_large_jpeg() {
 	input=$1
 	output=$2
 	xmp="$input.xmp"
-
+        xmp_profiling="$scriptdir/profiling-shot.xmp"
```

Comment:
This bash code (still) uses tabs for indentation, please stick to it

Reply:
If you fix that, we can merge the PR.

Reply:
Should be fixed now.